static/js/stats/stats.js

```diff
@@ -12,9 +12,34 @@ Plotly.register([PlotlyBar, PlotlyPie]);
 const font_14pt = {
     family: "Source Sans 3",
     size: 14,
-    color: "#000000",
+    color: "rgb(0,0,0)",
 };
 
+let button_color = "rgb(235,235,235)";
+
+let plot_color = "rgb(255,255,255)";
+
+//  function to set theme for stats page
+function setTheme(theme) {
+    if (theme === "dark-theme") {
+        document.querySelector("body").classList.remove("color-scheme-automatic");
+        document.querySelector("body").classList.add("dark-theme");
+        plot_color = "rgb(0,0,0)";
+        font_14pt.color = "rgb(221,222,238)";
+        button_color = "rgb(20,20,20)";
```

Comment:
How did you pick these colors?  I would prefer to use HSL like we do across the rest of the codebase if possible, and use a color that we already use in the web application for consistency.